homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -133,11 +148,13 @@ def __init__(self, coordinator, device_info, room_name, shade, name):
         """Initialize the shade."""
         super().__init__(coordinator, device_info, room_name, shade, name)
         self._shade = shade
+        self._attr_name = self._shade_name
         self._is_opening = False
         self._is_closing = False
         self._last_action_timestamp = 0
         self._scheduled_transition_update = None
-        self._current_hd_cover_position = MIN_POSITION
+        self._current_hd_cover_primary = MIN_POSITION
+        self._current_hd_cover_secondary = MIN_POSITION
```

Comment:
Having two sources of truth makes this hard to follow.

Now that you have `get_position_secondary` and `get_position_primary`, I think we can drop this